wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -409,4 +409,138 @@ function missiledefense_system_terms() {
 			printf( '%1$s', $html);
 		}
 	}
-endif;
\ No newline at end of file
+endif;
+
+if ( ! function_exists( 'missiledefense_actors_cat1' ) ) :
+	/**
+	 * Returns links to Category 1 Actors and their icons.
+	 *
+	 */
+	function missiledefense_actors_cat1() {
```

Comment:
Instead of having separate functions for the different categories of actors & defense systems, you could make this a single function with arguments.

`$group` is the field name.

```
missiledefense_actors('category_1_actors'); // call this in your `archive.php` file

function  missiledefense_actors($group) {
  // Other parts of the function here.
  $actors = get_field( $group, $object->name );
//  Other parts of the function here.
}

missiledefense_defsys_category('category_1_systems'); // call this in your `archive.php` file

function  missiledefense_defsys_category($group) {
  // Other parts of the function here.
  $systems = get_field( $group, $object->name );
//  Other parts of the function here.
}
```